doc/user/op_guide.md

```diff
@@ -2,9 +2,16 @@
 
 ## Install etcd operator
 
-Before you create the etcd-operator make sure you setup the necessary [rbac rules](./rbac.md) if your Kubernetes version is 1.7 or higher.
+For Kubernetes 1.7 or higher, set up [rbac rules](./rbac.md) for an
```

Comment:
This info is outdated.

> For Kubernetes 1.7 or higher

This is already mentioned in "Requirements" section in README. We should remove this.

>  set up rbac rules for an `etcd-operator` service account

This has changed. For simplicity, we use namespace default account.

The reason is that otherwise there would be many inconsistent parts in our examples, e.g. deployment.yaml. And we don't want to add to much complexity in open source etcd operator due to support capability. If a user cares about RBAC and using a different account, he needs to understand k8s RBAC and set up those himself.

Reply:
Maybe reduce this paragraph to only:

Setup RBAC rules for etcd-operator:
```
... run the script
```

Reply:
Applied. I've also added a version check to the script so if one attempts to run it on 1.6 or earlier, it will print something like: "Kubernetes version 1.6 predates RBAC support. Skipping role creation."